### PR TITLE
[docs] Add links to 'The Swift runtime' posts + podcast episode.

### PR DESCRIPTION
**Pull request by varungandhi-apple**

### CODE REVIEW

**Comment by varungandhi-apple** [comment]
@swift-ci please smoke test

File: docs/ExternalResources.md [hunk 1]
```diff
@@ -135,11 +135,24 @@ https://medium.com/kinandcartacreated/contributing-to-swift-part-2-efebcf7b6c93
 
 ## Code generation, runtime and ABI
 
-- The Swift Runtime ([Part 1: Heap Objects][], [Part 2: Type Layout][])
-  by Jordan Rose (blog post series, Aug-Sep 2020): This blog post series
-  describes the runtime layout for classes, closures and structs, as well
-  as basic runtime functionality such as retain/release that needs to be
-  handled when porting Swift to a new platform, such as [Mac OS 9][].
+- The Swift Runtime by Jordan Rose (blog post series, Aug-Oct 2020):
```
Comment (benrimmington) [inline docs/ExternalResources.md:138]:
```suggestion
- [The Swift Runtime][] by Jordan Rose (blog post series, Aug-Oct 2020):
```

File: docs/ExternalResources.md [hunk 2]
```diff
@@ -174,9 +187,15 @@ https://medium.com/kinandcartacreated/contributing-to-swift-part-2-efebcf7b6c93
   value witness tables, type metadata, abstraction patterns, reabstraction,
   reabstraction thunks and protocol witness tables.
 
+[Mac OS 9]: https://belkadan.com/blog/2020/04/Swift-on-Mac-OS-9/
```
Comment (benrimmington) [inline docs/ExternalResources.md:190]:
```suggestion
[Mac OS 9]: https://belkadan.com/blog/2020/04/Swift-on-Mac-OS-9/
[The Swift Runtime]: https://belkadan.com/blog/tags/swift-runtime/
```

**Review by benrimmington — APPROVED** [review]

**Comment by varungandhi-apple** [comment]
@swift-ci please smoke test

**Pull request merged**